### ISSUE TITLE
Uncaught exception when deleting an attachment

### ISSUE THREAD

**Issue by parapente** [jmcameron/attachments#90]
When deleting an attachment you get everytime an error like the above in the joomla everything.log:

```log
2024-12-04T18:31:40+00:00       CRITICAL 172.16.1.1     error   Uncaught Throwable of type Exception thrown with message "ΣΦΑΛΜΑ: Δεν είναι δυνατή η διαγραφή του συνημμένου: Δεν βρέθηκε έγκυρο ID συνημμένου (4)! (ERR 146)". Stack trace: #0 [ROOT]/libraries/src/MVC/Controller/BaseController.php(730): JMCameron\Component\Attachments\Administrator\Controller\AttachmentController->delete_warning()
#1 [ROOT]/libraries/src/Dispatcher/ComponentDispatcher.php(143): Joomla\CMS\MVC\Controller\BaseController->execute()
#2 [ROOT]/libraries/src/Component/ComponentHelper.php(361): Joomla\CMS\Dispatcher\ComponentDispatcher->dispatch()
#3 [ROOT]/libraries/src/Application/AdministratorApplication.php(150): Joomla\CMS\Component\ComponentHelper::renderComponent()
#4 [ROOT]/libraries/src/Application/AdministratorApplication.php(195): Joomla\CMS\Application\AdministratorApplication->dispatch()
#5 [ROOT]/libraries/src/Application/CMSApplication.php(306): Joomla\CMS\Application\AdministratorApplication->doExecute()
#6 [ROOT]/administrator/includes/app.php(58): Joomla\CMS\Application\CMSApplication->execute()
#7 [ROOT]/administrator/index.php(32): require_once('...')
#8 {main}
```

The attachment is indeed deleted so it is probably caused by a second deletion request for the same attachment that fails.

**Comment by JLTRY** [comment]
I reproduced this by deleting an attachments in admin article view

**Comment by parapente** [comment]
Fixed in PR #91
